Traducido Wizards al castellano.

4drian opened the pull request:
Me he tomado estas licencias al traducir:
> "Frost" se ha traducido como Gélido o Escarcha, dependiendo del contexto, para hacer los nombres mas legibles.
> "Fire" se ha traducido como Ígneo o Fuego, dependiendo del contexto, para hacer los nombres mas legibles.
> Para las traducciones de los hechizos, he buscado inspiracion en traducciones similares en WoW o DnD.

4drian committed:
Traducido al castellano.

Me he tomado estas licencias al traducir:
> "Frost" se ha traducido como Gélido o Escarcha, dependiendo del contexto, para hacer los nombres mas legibles.
> "Fire" se ha traducido como Ígneo o Fuego, dependiendo del contexto, para hacer los nombres mas legibles.
> Para las traducciones de los hechizos, he buscado inspiracion en traducciones similares en WoW o DnD.

4drian committed:
Agregadas las entradas de metadatos.
Corregido un error en on ice_lance.description.

nvb-uy commented on src/translations/wip.wizards.json:
```diff
@@ -1,96 +1,99 @@
-    "item.wizards.frost_spell_book": "Tome of Frost",
+    "item.wizards.arcane_spell_book": "Tomo Arcano",
+    "item.wizards.fire_spell_book": "Tomo Ígneo",
+    "item.wizards.frost_spell_book": "Tomo Gélido",
```
Preferiblemente seguir con la consistencia de usar Escharcha para Frost

4drian replied:
Vale, siendo mas literales, utilizare "de Fuego" y "de Escarcha" para "of Fire" y "of Frost".

Para mi gusto queda algo feo con los báculos mejorados "Báculo de Fuego de Rubí" porque usa mucho la preposición "de", pero creo que con el input que me das es el mejor compromiso.

nvb-uy commented on src/translations/wip.wizards.json:
```diff
@@ -1,96 +1,99 @@
+    "spell.wizards.fireball.name": "Bola de Fuego",
+    "spell.wizards.fireball.description": "Lanza una Bola explosiva de Fuego, causando hasta {damage} de hechizo de Fuego en un radio de {impact_range} bloques.",
+    "spell.wizards.fire_breath.name": "Aliento de Fuego",
+    "spell.wizards.fire_breath.description": "Incinera a los objetivos situados delante, causándoles hasta {damage} de hechizo de Fuego por segundo.",
```
Ejemplo de traducción:
"Hasta 18 de hechizo de fuego"

Cambiar a daño de fuego, en vez de simplemente hechizo de fuego.

4drian replied:
Corregido, ahora todo el daño se nombra de forma consistente para cada elemento. En todos los casos es ahora XX de daño de Elemento.

nvb-uy commented on src/translations/wip.wizards.json:
```diff
@@ -1,96 +1,99 @@
+    "spell.wizards.ice_lance.name": "Lanza de Escarcha",
+    "spell.wizards.ice_lance.description": "Lanza una esquirla de escarcha que atraviesa {pierce} objetivos, causando {damage} de hechizo de Escarcha al impactar.",
+    "spell.wizards.frostbolt.name": "Descarga Gélida",
+    "spell.wizards.frostbolt.description": "Lanza una bola de escarcha que rebota hacia {ricochet} objetivos cercanos adicionales, causando {damage} de hechizo de Escarcha y ralentizando al objetivo al impactar.",
```
Lo mismo,

"Causando 18 de hechizo de escarcha"

No suena muy bien in-game y puede ser confuso ya que no menciona específicamente que es daño

nvb-uy reviewed (changes requested):
Errores de consistencia, mayormente con el daño, y utilizando gélido e ígneo para los tomos de fuego y escarcha

nvb-uy commented:
Hice varios comentarios dando referencia de donde están los errores.

Actualmente ya hay otra persona trabajando en Wizards, Paladins y el resto de mods de RPG Series, pero con estos errores solucionados podría combinar ambas traducciones.

Fuera de eso, agradezco muchísimo tu contribución y me agrada mucho que este proyecto esté yendo bien

4drian committed:
Se realizan las siguientes correcciones para mantener la consistencia:
> Gélido se cambia a "de Escarcha"
> Ígneo se cambia a "de Fuego"
> Corregida la forma en la que se indica el daño.

4drian commented:
Cambios hechos, quedo a la espera por si hacen falta mas cambios.

A mi también me gusta el proyecto y espero que siga adelante. ¿Tenemos alguna lista de mods pendientes de traducir para no volver a solapar con otra persona?

nvb-uy commented on src/translations/wip.wizards.json:
```diff
@@ -83,16 +83,16 @@
-    "advancements.rpg_classes.obtain_fire_rune.description": "Consigue una Runa Ígnea",
-    "advancements.rpg_classes.obtain_frost_rune.title": "Senda Gélida",
-    "advancements.rpg_classes.obtain_frost_rune.description": "Consigue una Runa Gélida",
+    "advancements.rpg_classes.obtain_fire_rune.title": "Senda del Fuego",
```
Cambiar "Senda del ----" por "Camino Ígneo (Fuego) / Gélido (Escarcah) / Enigmático (Arcano)"

nvb-uy reviewed (changes requested):
Solo un cambio, el resto parece estar todo bien!

4drian committed:
Cambiado "Senda del xxx" por "Camino Ígneo/Gélido/Enigmático".

nvb-uy reviewed (approved):
Tengo unos cambios que me gustaría hacer pero los haré yo después del merge ^^
